.prettierrc.js

```diff
@@ -1,7 +0,0 @@
-module.exports = {
```

Comment:
just curious — no more prettier?

Reply:
prettier configuration moved to `package.json`.

Reply:
got it, package.json got into my blind spot :D